### ISSUE TITLE
Pre-work Checklist: Developer: Ashelle Cancio

### ISSUE THREAD

**Issue by arcan9** [hackforla/website#4066]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by arcan9** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings? 4.5 hours

**Comment by arcan9** [comment]
Progress report for the week of 03/06 - 03/11

I just completed my good first issue this past week and it got approved. I know the progress report is more for when we have spent five days on an issue, but so far I haven't spent five days on my issue. During the Sunday meeting today, it was mentioned that's it okay to comment my progress report here just to let HfLA know where I'm at.

**Comment by arcan9** [comment]
Progress report for the week of 03/12 - 03/18

My `good second issue` PR was completed, approved, and merged. ✅

PRs I reviewed:
- one (1) `good first issue` ✅
- two (2) `good second issue` ✅

Current issues I'm working on:
- Completed my `small` issue. Still waiting on approval. ⌚

**Comment by arcan9** [comment]
Progress report for week of 03/19:

Reviewed and/or currently reviewing a couple of issues. Review ETA: EOD 03/24.

Created PR for issue #4183. Waiting for approval and merge.

Availability: 3-8PM PST 03/24.

No blockers.

**Comment by ExperimentsInHonesty** [comment]
You are doing great!

**Comment by arcan9** [comment]
Progress for week of 03/27

Reviewed a PR

Blockers: Still waiting for PR #4274 to be approved.

Availability: after 5pm, M-F

**Comment by arcan9** [comment]
Progress for week of 04/02

Reviewed around 5 PRs

Currently working on my first large issue #1940. ETA on 04/14

Availability: after 6pm, M-F / afternoons on Sat

Blockers: I'm still waiting for my question to be answered, regarding my issue. Here: https://github.com/hackforla/website/issues/1940#issuecomment-1499747920

**Comment by arcan9** [comment]
Progress for week of 04/09

Availability: evenings M-F & afternoons on Sat

Been a little busy, but I'm still working on issue #1940. I'm reading through the wiki to see how to create a project board to test a GitHub action. Testing it successfully is probably my only blocker right now.

**Comment by arcan9** [comment]
Progress for week of 04/21

Unfortunately, I did not get anything done due to personal life issues.

I will get the PR for my large issue open for review soon.

**Comment by arcan9** [comment]
05/02

I will be taking a break from HfLA. I already talked to my dev lead about it, closed the PR I was working on, and unassigned myself from the issue.

**Comment by roslynwythe** [comment]
Hi @arcan9 I'm closing this issue but if you would like to return to Hack for LA in the future, you may reopen it and leave us an update.  Thank you.

**Comment by HackforLABot** [comment]
## Activity Log: arcan9
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- arcan9 opened: https://github.com/hackforla/website/issues/4066 at 02/28/2023, 08:08 PM PST
- arcan9 assigned: https://github.com/hackforla/website/issues/4066 at 02/28/2023, 08:24 PM PST
- arcan9 assigned: https://github.com/hackforla/website/issues/4026#issuecomment-1454306318 at 03/08/2023, 04:23 PM PST
- arcan9 commented on: https://github.com/hackforla/website/issues/4026#issuecomment-1461076990 at 03/08/2023, 04:25 PM PST
- arcan9 commented on: https://github.com/hackforla/website/issues/4066#issuecomment-1461078481 at 03/08/2023, 04:27 PM PST
- arcan9 opened: https://github.com/hackforla/website/pull/4139 at 03/08/2023, 10:36 PM PST
- arcan9 PR was merged: https://github.com/hackforla/website/pull/4139#event-8725152630 at 03/11/2023, 10:50 PM PST
- arcan9 issue closed by PR 4139: https://github.com/hackforla/website/issues/4026#event-8725152669 at 03/11/2023, 10:50 PM PST
- arcan9 commented on: https://github.com/hackforla/website/issues/4066#issuecomment-1465322151 at 03/12/2023, 03:56 PM PDT
- arcan9 assigned: https://github.com/hackforla/website/issues/4087#issuecomment-1465104338 at 03/12/2023, 04:11 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/issues/4087#issuecomment-1465326864 at 03/12/2023, 04:17 PM PDT
- arcan9 opened: https://github.com/hackforla/website/pull/4156 at 03/12/2023, 09:16 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4175#issuecomment-1467560222 at 03/14/2023, 12:43 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4175#pullrequestreview-1338758198 at 03/14/2023, 01:55 AM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4175#issuecomment-1468630062 at 03/14/2023, 11:34 AM PDT
- arcan9 PR was merged: https://github.com/hackforla/website/pull/4156#event-8751078721 at 03/14/2023, 10:07 PM PDT
- arcan9 issue closed by PR 4156: https://github.com/hackforla/website/issues/4087#event-8751078833 at 03/14/2023, 10:07 PM PDT
- arcan9 assigned: https://github.com/hackforla/website/issues/3756#issuecomment-1461226654 at 03/14/2023, 10:52 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/issues/3756#issuecomment-1469384105 at 03/14/2023, 10:54 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4202#issuecomment-1474405422 at 03/17/2023, 02:09 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4204#issuecomment-1474559081 at 03/17/2023, 06:13 PM PDT
- arcan9 opened: https://github.com/hackforla/website/pull/4205 at 03/17/2023, 07:07 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4202#pullrequestreview-1346869924 at 03/17/2023, 07:32 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/issues/4066#issuecomment-1474628085 at 03/17/2023, 07:46 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4204#pullrequestreview-1347348241 at 03/18/2023, 08:01 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4205#issuecomment-1475386199 at 03/19/2023, 01:20 PM PDT
- arcan9 PR was merged: https://github.com/hackforla/website/pull/4205#event-8790084115 at 03/19/2023, 09:25 PM PDT
- arcan9 issue closed by PR 4205: https://github.com/hackforla/website/issues/3756#event-8790084215 at 03/19/2023, 09:25 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4224#issuecomment-1478440424 at 03/21/2023, 12:05 PM PDT
- arcan9 assigned: https://github.com/hackforla/website/issues/4183 at 03/21/2023, 01:02 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/issues/4183#issuecomment-1478512492 at 03/21/2023, 01:05 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4224#pullrequestreview-1351434224 at 03/21/2023, 02:49 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4234#issuecomment-1480617035 at 03/22/2023, 10:19 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4259#issuecomment-1481454801 at 03/23/2023, 08:59 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4234#pullrequestreview-1355589496 at 03/23/2023, 01:51 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4259#pullrequestreview-1355765204 at 03/23/2023, 03:40 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4265#issuecomment-1482063212 at 03/23/2023, 04:58 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4265#pullrequestreview-1355829818 at 03/23/2023, 05:12 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4272#issuecomment-1482204942 at 03/23/2023, 08:40 PM PDT
- arcan9 opened: https://github.com/hackforla/website/pull/4274 at 03/23/2023, 10:08 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4267#issuecomment-1482267542 at 03/23/2023, 10:20 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/issues/4066#issuecomment-1482271480 at 03/23/2023, 10:27 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4275#issuecomment-1482991084 at 03/24/2023, 08:25 AM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4267#issuecomment-1483665943 at 03/24/2023, 06:33 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4272#issuecomment-1483666659 at 03/24/2023, 06:35 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4275#pullrequestreview-1357705697 at 03/24/2023, 06:57 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4275#pullrequestreview-1357797753 at 03/25/2023, 01:31 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4272#pullrequestreview-1357802058 at 03/25/2023, 01:56 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4267#pullrequestreview-1357803007 at 03/25/2023, 02:05 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4259#pullrequestreview-1357804040 at 03/25/2023, 02:15 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4288#pullrequestreview-1357863760 at 03/25/2023, 09:40 AM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4272#issuecomment-1483866658 at 03/25/2023, 09:43 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4272#pullrequestreview-1357864263 at 03/25/2023, 09:44 AM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4340#issuecomment-1488979710 at 03/29/2023, 10:05 AM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4274#issuecomment-1489111152 at 03/29/2023, 11:37 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4340#pullrequestreview-1366160173 at 03/30/2023, 07:21 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4274#issuecomment-1491211674 at 03/30/2023, 07:38 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/issues/4066#issuecomment-1491681595 at 03/31/2023, 03:13 AM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4359#issuecomment-1493050662 at 04/01/2023, 10:18 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4359#pullrequestreview-1367947858 at 04/01/2023, 02:58 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4274#issuecomment-1493137020 at 04/01/2023, 03:00 PM PDT
- arcan9 PR was merged: https://github.com/hackforla/website/pull/4274#event-8908289856 at 04/02/2023, 09:30 PM PDT
- arcan9 issue closed by PR 4274: https://github.com/hackforla/website/issues/4183#event-8908289940 at 04/02/2023, 09:30 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4386#issuecomment-1494990407 at 04/03/2023, 02:15 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4386#pullrequestreview-1372039371 at 04/04/2023, 07:20 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4408#issuecomment-1496833123 at 04/04/2023, 07:26 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4424#issuecomment-1496838781 at 04/04/2023, 07:35 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4424#pullrequestreview-1374012995 at 04/05/2023, 08:50 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4428#issuecomment-1499385801 at 04/06/2023, 10:21 AM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4430#issuecomment-1499390181 at 04/06/2023, 10:23 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4428#pullrequestreview-1375566933 at 04/06/2023, 01:57 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4430#pullrequestreview-1375572581 at 04/06/2023, 02:03 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4435#issuecomment-1499816892 at 04/06/2023, 06:18 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/issues/4066#issuecomment-1500717367 at 04/07/2023, 04:16 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4437#pullrequestreview-1376812998 at 04/08/2023, 03:40 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4437#issuecomment-1500991744 at 04/08/2023, 03:46 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4435#pullrequestreview-1376817446 at 04/08/2023, 04:59 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4453#issuecomment-1501187287 at 04/09/2023, 11:29 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4453#pullrequestreview-1377061121 at 04/09/2023, 09:19 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4435#pullrequestreview-1378119531 at 04/10/2023, 01:52 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4461#issuecomment-1503834970 at 04/11/2023, 10:45 AM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4461#pullrequestreview-1379937468 at 04/11/2023, 12:28 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4467#issuecomment-1503982303 at 04/11/2023, 12:31 PM PDT
- arcan9 opened: https://github.com/hackforla/website/issues/4488 at 04/11/2023, 08:15 PM PDT
- arcan9 submitted review: https://github.com/hackforla/website/pull/4467#pullrequestreview-1381876085 at 04/12/2023, 11:23 AM PDT
- arcan9 opened: https://github.com/hackforla/website/pull/4496 at 04/12/2023, 09:25 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/issues/4066#issuecomment-1509318954 at 04/14/2023, 02:49 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/issues/4066#issuecomment-1518428581 at 04/21/2023, 04:52 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/pull/4496#issuecomment-1532246207 at 05/02/2023, 03:43 PM PDT
- arcan9 closed: https://github.com/hackforla/website/pull/4496#event-9150386948 at 05/02/2023, 03:43 PM PDT
- arcan9 commented on: https://github.com/hackforla/website/issues/4066#issuecomment-1532249950 at 05/02/2023, 03:49 PM PDT
- arcan9 issue closed as completed: https://github.com/hackforla/website/issues/4066#event-9383710397 at 05/30/2023, 03:54 PM PDT
